packages/documentation-framework/components/tableOfContents/tableOfContents.js

```diff
@@ -117,7 +91,7 @@ export const TableOfContents = ({ items }) => {
     <JumpLinks
       label="Table of contents"
       isVertical
-      scrollableSelector=".pf-v6-c-page__main-container"
+      scrollableSelector="#ws-page-main"
       className="ws-toc"
       style={{ top: stickyNavHeight }}
       offset={width > 1450 ? 108 + stickyNavHeight : 148 + stickyNavHeight}
```

Comment:
Probably a new issue - but with the jumplinks collapsed, I notice that the scroll position puts the title underneath the jumplink toggle. The page switch looks 👌🏻 though!

Reply:
@srambach I'm only able to recreate if I navigate to a jumplink on desktop layout and then shrink the screen down so the jumplinks move from the side to the top as seen in the video, did you see it anywhere else?  Navigating from this point forward seems to work as expected at the new screen size.


https://github.com/user-attachments/assets/1ce8fa16-9645-467c-8a7a-d3a7cdb1c12b